src/IO/Importers/Actions/ReadVolumeData.hpp

```diff
@@ -5,6 +5,7 @@
 
 #include <cstddef>
```

Comment:
Can you split this commit? I'm not sure how many different things it's doing at once and am having a hard time following how to parse things... This should be at least 1 commit adding the selector enum, 1 commit using it, and possibly more commits changing target link libraries and `std::get`s.

Reply:
split the commits 👍